tools/slack/guidelines.md

```diff
@@ -50,7 +50,7 @@ See a list of other popular channels in the [Slack @ TTS](https://docs.google.co
 
 ## Usage tips
 
-- **[Add yourself to our custom emoji](https://gsa-tts.slack.com/customize/emoji).** Add your profile picture as a custom emoji with your name as the alias ("first-last"). This allows the whole TTS community to celebrate your contributions and serves as your introduction to our prolific custom emoji database. Post your emoji (and any other custom emoji you add) to the [#emoji-showcase](https://gsa-tts.slack.com/messages/C0X2T36AY) channel.
+- **[Add yourself to our custom emoji](#custom-emoji).** Add your profile picture as a custom emoji with your name as the alias ("first-last"). This allows the whole TTS community to celebrate your contributions and serves as your introduction to our prolific custom emoji database. Post your emoji (and any other custom emoji you add) to the [#emoji-showcase](https://gsa-tts.slack.com/messages/C0X2T36AY) channel.
```

Comment:
Maybe we should note here that this will put it in the approval queue on you should see your emoji added shortly. As a bit of an intro to the process, and so folks don't think it's immediate.